Sources/Kontur.Rabbitmq/AmqpProperties.cs

```diff
@@ -0,0 +1,53 @@
+﻿using RabbitMQ.Client;
+using System.Collections.Generic;
+
+namespace Kontur.Rabbitmq
+{
+    internal class AmqpProperties : IAmqpProperties
+    {
+        public string ReplyTo { get; set; }
+        public bool Persistent { get; set; }
+        public string MessageId { get; set; }
+        public IDictionary<string, string> Headers { get; set; }
+        public string CorrelationId { get; set; }
+        public string ContentType { get; set; }
+        public string ContentEncoding { get; set; }
+
+        public void CopyTo(IBasicProperties basicProperties)

```

Comment:
For what purposes do we need this function in AmqpProperties?

Reply:
A common message is converted to a RabbitMQ message in the Dataflow AmqpBuilderBlock. Then this message will be sent by the Dataflow AmqpSenderBlock. The AmqpBuilderBlock and the AmqpSenderBlock forms Dataflow pipeline. 
The problem is to create RabbitMQ message with using BasicProperties the AmqpBuilderBlock need to know the model (AMQP channel). But the model is accessible only in the AmqpSenderBlock.